lib/github-changelog.php

```diff
@@ -24,9 +31,69 @@ function make_github_request( $url, $headers = array() ) {
 	return json_decode( $data, true );
 }
 
+/**
+ * Wrapper which paginates the GitHub request to fetch everything. Useful when
+ * the endpoint doesn't include pagination information.
+ *
+ * @param string $url URL to get
+ * @return array $data The array with all the data
+ */
+function make_github_request_paginated( $url ) {
+	$all_data       = array();
+	$pagination_url = $url . '?per_page=50&page=';
```

Comment:
Should we bump this `per_page` to 100? 
GH API lets us return up to 100 items per page  ([#ref](https://docs.github.com/en/rest/pulls/pulls))- iff you set per_page=100, we will make fewer requests (not sure if we would have a scenario that would require that amount of requests though)